README.md

```diff
@@ -10,6 +10,13 @@ If your platform isn't yet supported, you can create the native binary from
 the latest upstream clang sources, make sure it is stripped and optimized
 (should be about 1.4MB as of mid-2015) and send a pull request to add it.
 
+## Globbing files
+
+    $ clang-format --glob=folder/**/*.js
+
+This will run `clang-format` once per file result, and show the total
```

Comment:
We need to document what the globbing semantics are. It's sufficient to point to the docs for the glob library we use.